app/Http/Controllers/TodoItemController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\TodoItem;
+use App\TodoList;
+
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+
+class TodoItemController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // 
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        // return view('todos.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request, TodoList $todoList)
+    {
+        // dd($request);
+        $rules = array('content' => ['required','unique:todo_items,content', 'max:255']);
+        $validator = Validator::make($request->all(), $rules);
+
+        if($validator->fails()){
+            return redirect()->route('todos.show')->withErrors($validator)->withInput();
+        } 
+        
+        $content = $request->input('content');
+        $item = new TodoItem();
+        $item->content = $content;
```

Comment:
@Teatoller You need to add the `todo_list_id` eg. `$item->todo_list_id = 1`. There is no default `todo_list_id` value to fall to for the `TodoItem` table.

Reply:
@bettblake08, 
Thank you for your insight.

Reply:
Hi bettblake08, 
The insight you provided enabled me to resolve the blocker. 
Thank you.